src/libs/Network/enhanceParameters.ts

```diff
@@ -1,27 +1,18 @@
-import lodashGet from 'lodash/get';
-import _ from 'underscore';
 import CONFIG from '../../CONFIG';
 import getPlatform from '../getPlatform';
 import * as NetworkStore from './NetworkStore';
 
 /**
  * Does this command require an authToken?
- *
- * @param {String} command
- * @return {Boolean}
  */
-function isAuthTokenRequired(command) {
-    return !_.contains(['Log', 'Authenticate', 'BeginSignIn', 'SetPassword'], command);
+function isAuthTokenRequired(command: string): boolean {
+    return !['Log', 'Authenticate', 'BeginSignIn', 'SetPassword'].includes(command);
```

Comment:
What about extracting `'Log', 'Authenticate', 'BeginSignIn', 'SetPassword'` strings to use across the files?

Reply:
Maybe let's do that in a follow-up PR? Together with better typings for API.write/API.read